.github/workflows/upgrade-testing.yaml

```
@@ -49,8 +49,8 @@ jobs:
       - name: K8GB deploy stable
         run: make deploy-stable
 
-      - name: K8GB upgrade candidate
-        run: make upgrade-candidate
+      - name: K8GB deploy semver
+        run: make deploy-semver
```

```suggestion
        run: make deploy-test-version
```
I suggest target renaming, please see below